app/Services/Manager.php

```diff
@@ -98,21 +98,36 @@ public function exportCSV($model, $hasReportback = false)
      * @param User $users
      * @return array $leaderboard
      */
-    public function getLeaderboard($competition)
+    public function getLeaderboard($competition, $ignoreFlagged = false)
     {
         $rows = [];
         $users = $competition->users;
 
         // Get all users in bulk
-        $users = $this->repository->getAll($users->pluck('id')->all());
-        $users = $users->keyBy('id')->all();
+        $ids = $users->pluck('id')->all();
+        $users = $this->repository->getAll($ids)->keyBy('id')->all();
```

Comment:
okie dokes, you should merge your stuff first then and ill resolve the conflicts